src/internal-links/handler.js

```diff
@@ -223,6 +214,53 @@ export const opportunityAndSuggestionsStep = async (context) => {
       kpiDeltas,
     },
   );
+
+  // For suggestions already marked as FIXED, if their urlTo is no longer 404,
```

Comment:
If a Suggestion's Fix entity is in Published state or we are moving it to Published state now, and we find that same issue again now or in a future audit, a new Suggestion should be created.

Please add any required handling for the same.
fwik, currently, we consider it as a Duplicate and ignore creation of a new suggestion and/or edit the existing suggestion